torch_xla/csrc/ops/max_pool2d_backward.cpp

```diff
@@ -0,0 +1,67 @@
+#include "ops/max_pool2d_backward.h"
+#include "lowering_context.h"
+#include "ops/infer_output_shape.h"
+#include "pooling.h"
+#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
+#include "tensorflow/compiler/xla/xla_client/util.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::Shape NodeOutputShape(
+    const Value& grad_output, const Value& input,
+    tensorflow::gtl::ArraySlice<const xla::int64> kernel_size,
+    tensorflow::gtl::ArraySlice<const xla::int64> stride,
+    tensorflow::gtl::ArraySlice<const xla::int64> padding) {
+  auto lower_for_shape_fn =
+      [stride, padding,
+       kernel_size](tensorflow::gtl::ArraySlice<const xla::XlaOp> operands)
+      -> xla::XlaOp {
+    XLA_CHECK_EQ(operands.size(), 2)
+        << "Unexpected number of operands: " << operands.size();
```

Comment:
IMO we can get rid of these extra string messages.
That macros already provides a text like:

"Error: XLA_CHECK_EQ(operands.size(), 2) : 15 vs 2"

Which is a pretty clear interpretation even w/out the extra message.

Reply:
Done.